playbooks/openstack/resources.py

```diff
@@ -19,6 +19,13 @@
 from keystoneauth1.exceptions.catalog import EndpointNotFound
 import shade
 
+# `OPENSHIFT_CLUSTER` env variable is deprecated and should be removed
+# in further releases. Please use `OPENSHIFT_STACK_NAME` instead of.
+OPENSHIFT_CLUSTER = os.getenv('OPENSHIFT_CLUSTER', 'openshift-cluster')
+
+OPENSHIFT_STACK_NAME = os.getenv('OPENSHIFT_STACK_NAME', OPENSHIFT_CLUSTER)
+OPENSHIFT_CLUSTER_ID = os.getenv('OPENSHIFT_CLUSTER_ID')
```

Comment:
Would it be possible to reduce these three into a single environment variable?

Reply:
I think yes, but it'll confuse people since these variables are related to different configuration options in openshift ansible plugin, that's why i decided to not use only OPENSHIFT_CLUSTER.

Reply:
OPENSHIFT_CLUSTER exists precisely for this purpose. I'd very much prefer if you used it.